packages/shared/src/components/fields/MarkdownInput/CommentMarkdownInput.tsx

```diff
@@ -129,6 +133,12 @@ export function CommentMarkdownInput({
 
     if (!editCommentId) {
       updatePostCache(client, postId, { numComments: post.numComments + 1 });
+
+      trackEvent(
```

Comment:
Does this mean we missed some tracking events before from companion?

Reply:
Only after the migration we did from Modal to CommentBox.

Reply:
Why don't we send this event from the component itself? The fact we have to add everywhere is a bit messy

Reply:
Yes, this is the fix now, we are sending it from the component itself than the callback.

Reply:
@idoshamun Just mentioned you as I'm not sure data team needs to be aware of these things?

Reply:
Thanks. Since we don't have so many companion users it's fine :)